applications/milliwatt/src/milliwatt_listener.erl

```diff
@@ -26,10 +26,7 @@
 -record(state, {}).
 -type state() :: #state{}.
 
--define(BINDINGS, [{'route', [{'types', ?RESOURCE_TYPES_HANDLED}
-                             ,{'restrict_to', ['no_account']}
```

Comment:
well, karl said to have milliwatt respond to unauthenticated calls, and that's why this is here